cmd/agent/app/reporter/flags.go

```diff
@@ -52,45 +52,6 @@ func AddFlags(flags *flag.FlagSet) {
 // InitFromViper initializes Options with properties retrieved from Viper.
 func (b *Options) InitFromViper(v *viper.Viper) *Options {
 	b.ReporterType = Type(v.GetString(reporterType))
-	b.AgentTags = parseAgentTags(v.GetString(agentTags))
+	b.AgentTags = flags.ParseJaegerTags(v.GetString(agentTags))
 	return b
 }
-
-// Parsing logic borrowed from jaegertracing/jaeger-client-go
-func parseAgentTags(agentTags string) map[string]string {
```

Comment:
Note that this package currently has 100% coverage
```
$ go test -cover ./cmd/agent/app/reporter/
ok  	github.com/jaegertracing/jaeger/cmd/agent/app/reporter	0.069s	coverage: 100.0% of statements
```
which means there is a test covering this function, which should also move to the new location.

Reply:
I am not sure if moving this test out of there is a good idea? It does what it should be doing, parses agent options with tags. I've added additional tests to `cmd/flags`. I could remove `TestBindFlags` test from `cmd/agent/app/reporter/flags_test.go` but I think that would leave the actual option handling without coverage?

Moving the complete test from `cmd/agent/app/reporter/flags_test.go` to `cmd/flags/flags_test.go` does not seem to be an option as is results in circular import.